pkgs/development/python-modules/matplotlib/2.nix

```diff
@@ -43,7 +43,9 @@ buildPythonPackage rec {
 
   buildInputs = [ which sphinx ]
     ++ stdenv.lib.optional enableGhostscript ghostscript
-    ++ stdenv.lib.optional stdenv.isDarwin [ Cocoa ];
+    ++ stdenv.lib.optional stdenv.isDarwin Cocoa
+    ++ stdenv.lib.optional enableQt wrapQtAppsHook
```

Comment:
Here you add to `buildInputs`, but for matptlotlib 3.x it goes to `nativeBuildInputs`

Reply:
Right, it should go to `nativeBuildInputs`.

Reply:
Not resolved.

Reply:
Thank you, I messed up again when rebasing. I should be more careful when working from multiples computers.